### PR TITLE
Release 3.2.1

### PR DESCRIPTION
**Pull request by trevorsibanda**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -7,8 +7,9 @@ title: Documentation
 
 #### Releases
 
-Latest: [3.2.0](/doc/3.2.0/)
+Latest: [3.2.1](/doc/3.2.1/)
 
+* [3.2.1](/doc/3.2.1/)
 * [3.2.0](/doc/3.2.0/)
```
Comment (szeiger) [inline docs/index.md]: These two should be on one line (like the 3.1.x and 3.0.x lines below)

**Comment by cvogt** [comment]
LGTM

File: news/_posts/2017-07-20-slick-3.2.1-released.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  the codebase.
+* [``3728a2092``](https://github.com/slick/slick/commit/3728a2092d78afd53b54d0b9f6cfdf29af1fb3c5) regression: final actually breaks compilation
+* [``3f5c6753b``](https://github.com/slick/slick/commit/3f5c6753b4108564a3583c8b49f0a89374a6dd4c) fixing part 1 of #1408, flatmaps in the tail position of a query are unrolled into nested bind clauses* Strip margins of multi-line SQL interpolator strings.
+* [``8b342b4a1``](https://github.com/slick/slick/commit/8b342b4a177770431223c2a2559041a009eda9f3) Integrate the four official Activator templates into the manual.
+* [``ddb75a0f9``](https://github.com/slick/slick/commit/ddb75a0f96fdde48bbb6885c4636bae204515a69) Fix ambiguity for MappedProjection Shape
+* [``21045bbf2``](https://github.com/slick/slick/commit/21045bbf2df79f18feabe0c147dfb0dc202f86a9) use mix of Java/scala reflection to retrieve inUseCount on 2.12.x
+
+
+You can find details about upgrading from Slick 3.2 in the *Migration Guide*.
```
Comment (szeiger) [inline news/_posts/2017-07-20-slick-3.2.1-released.md]: from 3.1

File: news/_posts/2017-07-20-slick-3.2.1-released.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+* [``21045bbf2``](https://github.com/slick/slick/commit/21045bbf2df79f18feabe0c147dfb0dc202f86a9) use mix of Java/scala reflection to retrieve inUseCount on 2.12.x
+
+
+You can find details about upgrading from Slick 3.2 in the *Migration Guide*. 
+
+This release contains contributions by the following committers since 3.2 (in alphabetical order):
+
+[astorije](https://github.com/astorije), [cvogt](https://github.com/cvogt) , [deusaquilus](https://github.com/deusaquilus), [kuppuswamy](https://github.com/kuppuswamy), [kwark](https://github.com/kwark), [njouanin](https://github.com/njouanin), [olafurpg](https://github.com/olafurpg), [stewSquared](https://github.com/stewSquared), [szeiger](https://github.com/szeiger), [trevorsibanda](https://github.com/trevorsibanda)
+
+##### Here is a complete list of changes since 3.2
```
Comment (szeiger) [inline news/_posts/2017-07-20-slick-3.2.1-released.md]: should probably say 3.2.0 here (and 4 lines up, too)

File: news/_posts/2017-07-20-slick-3.2.1-released.md (hunk 2 above)
Comment (szeiger) [inline news/_posts/2017-07-20-slick-3.2.1-released.md]: ...oh, and it's called *Upgrad Guide* in the actual manual. If you search for "migration" you get something about database migrations.

**Pull request merged**